src/XRP/default-xrp-client.ts

```diff
@@ -43,14 +38,14 @@ import { LedgerSpecifier } from './Generated/web/org/xrpl/rpc/v1/ledger_pb'
 import SendXrpDetails from './model/send-xrp-details'
 import { AccountSetFlag } from './model/account-set-flag'
 import TransactionResult from './model/transaction-result'
-
-/** A margin to pad the current ledger sequence with when submitting transactions. */
-const maxLedgerVersionOffset = 10
+import CommonXrplClient from './common-xrpl-client'
 
 /**
  * DefaultXrpClient is a client which interacts with the XRP Ledger.
```

Comment:
is there a more specific comment we can update this to? this is the same comment for CommonXrplClient

Reply:
yep, updated both